sync/tc.py

```diff
@@ -443,3 +444,38 @@ def cleanup():
                 now - timedelta(days=5)):
                 logger.info("Removing downloaded logs without recent activity %s" % rev_path)
                 shutil.rmtree(rev_path)
+
+
+def get_wpt_report(tasks, pr_id, session=None):
```

Comment:
Historically this kind of code hasn't been in tc.py itself (e.g. the existing code is in the try class). Could be a chance to refactor somewhat, so that we can share codepaths for downloading logs with the try case.

Reply:
I have been toying with the idea of making a TryPush like class for these Taskcluster runs, will look into how this could work